xchange-binance/src/test/java/org/knowm/xchange/binance/service/account/AccountServiceResilienceTest.java

```diff
@@ -0,0 +1,52 @@
+package org.knowm.xchange.binance.service.account;
+
+import org.junit.Test;
+import org.knowm.xchange.binance.AbstractResilienceTest;
+import org.knowm.xchange.binance.BinanceExchange;
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.exceptions.ExchangeSecurityException;
+import org.knowm.xchange.service.account.AccountService;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+public class AccountServiceResilienceTest extends AbstractResilienceTest {
```

Comment:
you are not testing any resiliency features here, please do not extend AbstractResilienceTest 

change the test name to `AccountServiceTest `

and setup you own WireMock here

Reply:
done